src/payment-account.js

```diff
@@ -13,6 +16,68 @@ class PaymentAccount extends ExchangePaymentAccount {
       return trade;
     });
   }
+
+  sell () {
+    console.log('called sell', this);
+    const sellData = {
+      priceQuoteId: this._quote._id,
+      transferIn: {
+        medium: 'blockchain'
+      },
+      transferOut: {
+        medium: 'bank',
+        mediumReceiveAccountId: this._account.id
+      }
+    };
+    console.log('sell payload', sellData);
+    return this._api.authPOST('trades', sellData);
+  }
+
+  static add (api, obj) {
+    const b = {
+      account: {
+        currency: obj.account.currency,
+        bic: obj.account.bic,
+        number: obj.account.number
+      },
+      holder: {
+        name: obj.holder.name,
+        address: {
+          street: obj.holder.address.street,
+          city: obj.holder.address.city,
+          zipcode: obj.holder.address.zipcode,
+          country: obj.holder.address.country
+        }
+      },
+      bank: {
+        // name: obj.bank.name || null,
+        address: {
+          country: obj.bank.address.country,
+          street: obj.bank.address.street || null,
+          zipcode: obj.bank.address._ipcode || null,
+          city: obj.bank.address.city || null
+        }
+      }
+    };
+    return api.authPOST('bank-accounts', b).then(res => {
+      return new PaymentAccount(api, undefined, undefined, res);
+    });
+  }
+
+  static getAll (api, quote) {
+    return api.authGET('bank-accounts').then((accounts) => {
+      let accountsObj = [];
+      for (let account of accounts) {
+        accountsObj.push(new PaymentAccount(api, undefined, quote, account));
+      }
+      return accountsObj;
+    });
+  }
+
+  deleteOne (api) {
```

Comment:
Might be better to add an instance method `delete`, so one can call `accounts[0].delete()`